series/supportedseries.go

```diff
@@ -110,17 +112,20 @@ var ubuntuSeries = map[string]string{
 	"xenial":  "16.04",
 	"yakkety": "16.10",
 	"zesty":   "17.04",
+	"artful":  "17.10",
+	"bionic":  "18.04",
 }
 
-// ubuntuLts provides a lookup for current LTS series.  Like seriesVersions,
+// ubuntuLTS provides a lookup for current LTS series.  Like seriesVersions,
 // the values here are current at the time of writing. On Ubuntu systems this
 // map is updated by updateDistroInfo, using data from
 // /usr/share/distro-info/ubuntu.csv to ensure we have the latest values.  On
 // non-Ubuntu systems, these values provide a nice fallback option.
-var ubuntuLts = map[string]bool{
+var ubuntuLTS = map[string]bool{
 	"precise": true,
 	"trusty":  true,
 	"xenial":  true,
+	// TODO(rogpeppe) add bionic when it's released.
```

Comment:
April 26th according to https://wiki.ubuntu.com/BionicBeaver/ReleaseSchedule, not sure if that's worth adding to the comment.

Reply:
I'll leave it for now.